composer.json

```diff
@@ -22,7 +22,10 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
Do we have a better solution for this?

Reply:
We'll need to keep this, as it is needed for `composer/composer:^2.2`, and will be mandatory starting mid-2022